app/src/main/java/org/fossasia/susi/ai/chat/ChatPresenter.kt

```diff
@@ -53,6 +53,7 @@ class ChatPresenter(chatActivity: ChatActivity) : IChatPresenter, IChatModel.OnR
     var backPressedOnce = false
     var id: Long = 0;
     var identifier: String = ""
+    var tableDatas: TableDatas? = null
```

Comment:
data itself is a plural. Please change datas to data.

Reply:
TableDatas is the name of the class and also if these changes don't matter a lot can we merge this PR as we are behind the release deadline

Reply:
I don't know if it is right to use a non-existent word in dictionary to name a variable or a class. Once it gets merged nobody is going to change it, and it will stay there in the code.
@iamareebjamal If this is fine for you, then rest LGTM.

Reply:
@arundhati24 as said above this bit needs refactoring and I said I will take care of them so these spell checks can be ignored, also naming variables is subjective if it looks good please approve and merge this PR.

Reply:
@arundhati24 the TableData class is a different class and I don't know why is this such a big deal

Reply:
Yes, TableDatas does not feel right, I'm going to say that you can continue to go with this if you send the refactoring PR today itself

Reply:
Yes @iamareebjamal I will refactor PR by today